docs/troubleshooting.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[troubleshooting]]
+== Troubleshooting
+
+Is something not working as expected?
+Don't worry if you can't figure out what the problem is; we’re here to help!
+As a first step, ensure your app is compatible with the Agent's <<supported-tech,supported technologies>>.
+
+If you're an existing Elastic customer with a support contract, please create a ticket in the
+https://support.elastic.co/customers/s/login/[Elastic Support portal].
+Other users can post in the https://discuss.elastic.co/c/apm[APM discuss forum].
+
+IMPORTANT: *Please attach your debug logs* so that we can analyze the problem.
```

Comment:
I think we should call out here that the Go agent doesn't have any logging by default; you need to enable it by setting [ELASTIC_APM_LOG_FILE](https://www.elastic.co/guide/en/apm/agent/go/current/configuration.html#config-log-file). Perhaps add another section below?

Reply:
Added!